### PR TITLE
Update Automation Servers Pages

### PR DESCRIPTION
**Pull request by jamestalton**
- Adds support for new automation server type - Galaxy - needed by Hub PM for demoing new look
- Updates Automation Servers to be stored in IndexDb instead of local storage - will allow for more consistent handling of automation servers,
- Adds new setting UI_MODE for stand alone modes
- Fixed the proxy to handle when no port is specified

### CODE REVIEW

File: frontend/automation-servers/AutomationServerSwitcher.tsx [hunk 1]
```diff
@@ -7,15 +7,16 @@ import {
...
+import { useAutomationServerTypes } from './hooks/useAutomationServerTypes';
 
 export function AutomationServerSwitcher() {
-  const { t } = useTranslation();
-  const { automationServer, automationServers } = useAutomationServers();
+  const automationServers = useAutomationServers();
+  const automationServer = useActiveAutomationServer();
+  const automationServerTypes = useAutomationServerTypes();
   const [open, setOpen] = useState(false);
+  // const automationServerTypes = useAutomationServerTypes();
```
Comment (keithjgrant) [inline frontend/automation-servers/AutomationServerSwitcher.tsx:19]: delete commented code

File: frontend/automation-servers/IndexDb.tsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+      const itemMap = itemMapRef.current;
+      const lastModified = lastModifiedRef.current;
+      const newItems = await db.getAllFromIndex(
+        storeName,
+        'modified',
+        IDBKeyRange.lowerBound(lastModified.lastModified, true)
+      );
+      if (abortSignal.aborted) return;
+      if (newItems && newItems.length > 0) {
+        // console.log(storeName, newItems)
```
Comment (keithjgrant) [inline frontend/automation-servers/IndexDb.tsx:147]: delete commented line

File: frontend/automation-servers/hooks/useAutomationServerFilters.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        placeholder: t('Filter by server type'),
+        options: options,
+      },
+      // {
+      //   key: 'labels',
+      //   type: ToolbarFilterType.Text,
+      //   query: 'labels',
+      //   label: t('Label'),
+      //   comparison: 'contains',
+      // },
```
Comment (keithjgrant) [inline frontend/automation-servers/hooks/useAutomationServerFilters.tsx:33]: delete

File: frontend/automation-servers/hooks/useAutomationServersColumns.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      {
+        header: t('Url'),
+        value: (server) => server.url,
+        cell: (server) => <CopyCell text={server.url} />,
+      },
+      // {
+      //   header: t('Labels'),
+      //   type: 'labels',
+      //   value: (server) => server.labels,
+      // },
```
Comment (keithjgrant) [inline frontend/automation-servers/hooks/useAutomationServersColumns.tsx:41]: .

**Review by keithjgrant — APPROVED** [review]

**Comment by himdel** [comment]
How is the hub API prefix supposed to work now?

The point of #606 was to make this default to `/api/automation-hub/` and overridable using `HUB_API_BASE_PATH`.

This PR seems to change the default to an empty string, and add some logic depending on `activeAutomationServer?.type`.

How/why should the server type affect galaxy urls? 
Why the emtpy default, does it get used with some other set of variables so it's never empty?

**Pull request merged**